app/controllers/api/v1/rooms_configurations_controller.rb

```diff
@@ -3,6 +3,10 @@
 module Api
   module V1
     class RoomsConfigurationsController < ApiController
+      before_action only: %i[index] do
+        ensure_authorized(%w[CreateRoom ManageSiteSettings ManageRoles], friendly_id: params[:friendly_id])
+      end
```

Comment:
A user with a SharedRoom wouldn't be able to change the settings of a SharedRoom because it's neither it's own room, nor does he has the ManageSiteSettings permission.

Reply:
Ah, I didn't know that.
Yeah, for now it will crash if the user is going to the Room settings